README.md

```diff
@@ -46,4 +46,39 @@ Configuration guide is available at https://github.com/DeloitteDigitalAT/core-ba
 
 - Sonar
 
-- TBD
\ No newline at end of file
+- TBD
+
+## How to run services and db locally. Requires JDK 11+ and Docker
+1) Build containers
+```
+cd data-access-services
+mvn install
+./docker-build.sh
+
+cd ../domain-service/credit-application/
+./docker-build.sh
+
+cd ../data-service
+./docker-build.sh
+```
+
+2) Run docker compose
+From root directory:
```

Comment:
its moved to `devops/docker` dir.

Reply:
changed instruction. Added 'cd devops/docker/'